cpu/samd5x/cpu.c

```diff
@@ -24,75 +24,75 @@
 #include "macros/units.h"
 #include "periph_conf.h"
 #include "periph/init.h"
-#include "stdio_base.h"
 
 /*
  * An external inductor needs to be present on the board,
  * so the feature can only be enabled by the board configuration.
  */
 #ifndef USE_VREG_BUCK
-#define USE_VREG_BUCK   (0)
+#  define USE_VREG_BUCK (0)
```

Comment:
Is this type of indenting documented somewhere in the coding convention. I couldn't find it having a brief look at the RIOT and Linux one.

I'm asking because it's the first time seeing this, I would have expected indenting the `#` as well. Grepping through the code base it seems we currently have all three of:

```c
#if
#define
#endif

#if
#  define
#endif

#if
    #define
#endif
```

Reply:
Interesting to know.

I agree that it increases readability. If we want to go with this in general, we should maybe include it in the coding convention, too?

Reply:
Good idea to include it in the coding convention. I thought it actually was. Having such things explicitly written makes it a lot more transparent and accessible.

Reply:
follow-up in #20866